.github/workflows/golang.yaml

```diff
@@ -30,54 +30,73 @@ jobs:
     steps:
     - uses: actions/checkout@v4
       name: Checkout code
+
     - name: Get Golang version
       id: vars
       run: |
         GOLANG_VERSION=$(./hack/golang-version.sh)
         echo "GOLANG_VERSION=${GOLANG_VERSION##GOLANG_VERSION := }" >> $GITHUB_ENV
+
     - name: Install Go
       uses: actions/setup-go@v5
       with:
         go-version: ${{ env.GOLANG_VERSION }}
+
     - name: Lint
       uses: golangci/golangci-lint-action@v8
       with:
         version: latest
         args: -v --timeout 5m
         skip-cache: true
+
     - name: Check golang modules
       run: | 
         make check-vendor
         make -C deployments/devel check-modules
+
   test:
     name: Unit test
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+
       - name: Get Golang version
         id: vars
         run: |
           GOLANG_VERSION=$(./hack/golang-version.sh)
           echo "GOLANG_VERSION=${GOLANG_VERSION##GOLANG_VERSION := }" >> $GITHUB_ENV
+
       - name: Install Go
         uses: actions/setup-go@v5
         with:
           go-version: ${{ env.GOLANG_VERSION }}
-      - run: make test
+      
+      - name: Generate coverage report
+        run: make coverage
+
+      - name: Upload to Coveralls
+        uses: coverallsapp/github-action@v2
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          path-to-lcov: coverage.out
```

Comment:
Should this just be `file:`? https://github.com/coverallsapp/github-action?tab=readme-ov-file#inputs

Reply:
Does `file` work?

Reply:
We may want to make a similar change to the GPU Operator and `k8s-operator-libs`. cc @tariq1890